examples/notebooks/healthcare/beam_post_hl7_messages_to_hcapi.ipynb

```diff
@@ -0,0 +1,504 @@
+      "cell_type": "markdown",
+      "source": [
+        "# **Post Hl7v2 messages to Google Cloud Healthcare API HL7v2 store pipeline**\n",
+        "\n",
+        "This example demonstrates how to set up an Apache Beam pipeline that reads a HL7 file from [Google Cloud Storage](https://https://cloud.google.com/storage), and calls the [Google Cloud Healthcare API Hl7v2 store to store Hl7 messages](https://cloud.google.com/healthcare-api/docs/how-tos/hl7v2-messages) to extract information from unstructured data. This application can be used in contexts such as reading raw Hl7 messages, if needed parse them or modify them as per your defined Hl7v2 store configurations and store data into Hl7v2 store.\n",
+        "\n",
+        "An Apache Beam pipeline is a pipeline that reads input data, transforms that data, and writes output data. It consists of PTransforms and PCollections. A PCollection represents a distributed data set that your Beam pipeline operates on. A PTransform represents a data processing operation, or a step, in your pipeline. It takes one or more PCollections as input, performs a processing function that you provide on the elements of that PCollection, and produces zero or more output PCollection objects.\n",
+        "\n",
+        "For details about Apache Beam pipelines, including PTransforms and PCollections, visit the [Beam Programming Guide](https://beam.apache.org/documentation/programming-guide/).\n",
+        "\n",
+        "You'll be able to use this notebook to explore the data in each PCollection."
+      ],
+      "metadata": {
+        "id": "wC9KRrlORwKu"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "***Sample HL7v2 Message***\n",
+        "\n",
+        "The below reference message shows a sample Hl7v2 messages seperated by \\r.\n",
+        "\n",
+        "**MSH|^~\\&|FROM_APP|FROM_FACILITY|TO_APP|TO_FACILITY|20150503223000||ADT^A01|20150503223000|P|2.5|\\r\n",
+        "EVN|A01|20110613083617|\\r\n",
+        "PID|1||21004053^^^^MRN||SULLY^BRIAN||19611209|M|||123 MAIN ST^^MOUNTAIN SPRINGS^CO^80439|\\r\n",
+        "PV1||I|H73 RM1^1^^HIGHWAY 73 CLINIC||||5148^MARY QUINN|||||||||Y||||||||||||||||||||||||||||20150503223000|**\n",
+        "\n",
+        "The file contains many such messages and the objective of this code will be to split and construct messages and POST it to Google Cloud HealthCare API HL7v2 store."
+      ],
+      "metadata": {
+        "id": "AOVYgtyaqSxa"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Lets install necessary packages"
+      ],
+      "metadata": {
+        "id": "81wCK9XnS6Sc"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!pip install apache-beam[gcp]"
+      ],
+      "metadata": {
+        "id": "Yv1phmRZS23c"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "**GCP Setup**\n",
```

Comment:
I think we'd benefit from a brief intro sentence explaining why we need GCP creds and how you can set up an account if you don't have one.

Reply:
Added the GCP account creation info.